exporter/loadbalancingexporter/resolver_static.go

```diff
@@ -0,0 +1,75 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loadbalancingexporter
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/tag"
+)
+
+var _ resolver = (*staticResolver)(nil)
+
+var errNoEndpoints = errors.New("no endpoints specified for the static resolver")
+
+type staticResolver struct {
+	endpoints         []string
+	onChangeCallbacks []func([]string)
+	once              sync.Once // we trigger the onChange only once
+}
+
+func newStaticResolver(endpoints []string) (*staticResolver, error) {
+	if len(endpoints) == 0 {
+		return nil, errNoEndpoints
+	}
+
+	return &staticResolver{
+		endpoints: endpoints,
+	}, nil
+}
+
+func (r *staticResolver) start(ctx context.Context) error {
+	r.resolve(ctx) // right now, this can't fail
+	return nil
```

Comment:
Does this mean that is onChange is called after start the callback will never be called? Is this expected behavior?

Reply:
Yes, for the static resolver, this is the expected behavior. There's no need to continuously resolve the same list of static hosts.

Reply:
Please document the behavior.